pkg/core/core.go

```diff
@@ -20,9 +20,10 @@ const (
 )
 
 type Flags struct {
-	DamageMode   bool
-	LogDebug     bool
-	ChildeActive bool
+	DamageMode    bool
+	LogDebug      bool // Used to determine logging level
+	ChildeActive  bool // Used for Childe +1 NA talent passive
+	OHCActiveChar int  // Shows which character currently has an active OHC proc. -1 = Non-active
```

Comment:
Should ideally use custom flags for this instead of adding to the flags struct: https://github.com/genshinsim/gcsim/blob/0a1394e66d5b675ad6ba6ebbbe0dacf2612c87df/pkg/core/core.go#L221

Arguably I should have done the same for childe flag though

Reply:
I can make the Childe change in another bugfix PR